### DISCUSSION TITLE
Using “placeholder” attribute and “aria-placeholder” both together

### DISCUSSION THREAD

**Discussion by Accessibility-SME** [w3c/wcag#3136] [Q&A]
Can we use “placeholder” attribute and “aria-placeholder” both together? so that browser and screen reader that does not support placeholder and read using aria-placeholder content?

**Comment by JAWS-test** [comment]
It is not forbidden to use ARIA and HTML attributes in parallel. However, it should be noted that in most cases the HTML attribute takes precedence in case of conflicts (in case of a checkbox with checked aria-checked=false, the ARIA attribute aria-checked is ignored). 
Also, it should be noted that placeholder is not so great for input hints, since it is not permanently visible. It would be better to display the input notes next to or below the field and link them to the field via aria-describedby.
I want to emphasize that I don't know of any browser or screen reader that supports aria-placeholder better than placeholder.
